ruff.toml

```diff
@@ -46,10 +46,10 @@ ignore = [
 
 [lint.extend-per-file-ignores]
 # Ignore missing docstrings and type annotations for selected directories
-"./*.py" = ["D101", "D102", "D103", "ANN"]
-"tests/*" = ["D101", "D102", "D103", "ANN"]
-"examples/**/*" = ["D101", "D102", "D103", "ANN"]
-"tesseract_core/sdk/templates/*" = ["D101", "D102", "D103", "ANN"]
+"./*.py" = ["D101", "D102", "D103", "D106", "ANN"]
```

Comment:
Some esoteric lint error about missing docstrings in nested classes...

Reply:
Couldn't we just add docstrings to said public nested classes, or mark them as private?

Reply:
we could, but why enforce this if not even regular docstrings are enforced?